packages/blocks/divider/src/App.tsx

```diff
@@ -2,6 +2,15 @@ import React from "react";
 
 import { BlockComponent } from "blockprotocol/react";
 
-type AppProps = {};
+type AppProps = { color?: string; height?: string | number };
 
-export const App: BlockComponent<AppProps> = () => <hr />;
+export const App: BlockComponent<AppProps> = ({ color, height }) => (
+  <hr
+    style={{
+      width: "100%",
+      border: "none",
+      backgroundColor: color ?? "black",
+      height: height ?? "1px",
```

Comment:
I vaguely remember we had issues with `??` syntax. Can it be still true for `yarn build`?

Reply:
It has more to do with the target version set inside `tsconfig.json` I think, although I don't have context on the issue you're describing here.